packages/core/src/lib/input/range-input.svelte

```diff
@@ -0,0 +1,214 @@
+<!--
+@component
+  
+For tweakable numeric user inputs that have minimum and maximum values.
+
+```svelte
+<RangeInput max={1} step={0.01} />
+```
+-->
+<svelte:options immutable />
+
+<script lang="ts">
+import { writable } from 'svelte/store';
+import cx from 'classnames';
+
+import { preventHandler, preventKeyboardHandler } from '$lib/prevent-handler';
+import NumericInput from './numeric-input.svelte';
+import { uniqueId } from '$lib/unique-id';
+
+/** The minimum allowed value. */
+export let min = 0;
+
+/** The maximum allowed value. */
+export let max = 100;
+
+/** The amount to add/subtract when moving the slider. */
+export let step = 1;
+
+/** The value of the input, defaults to the value of `min` */
+export let value: number = min;
+
+/** Whether or not the input should be rendered as readonly and be operable. */
+export let readonly = false as boolean | undefined;
```

Comment:
Why cast to undefined? Could we not just use the false default? (for `disabled` as well)

Reply:
I was following what we do in the `Input` component. I am guessing there was some linting issues previously, I removed the cast and it is working fine. Going to go ahead and clean that up elsewhere.